httpserver/lmq_server.cpp

```diff
@@ -220,6 +294,12 @@ void LokimqServer::init(ServiceNode* sn, RequestHandler* rh,
     lokimq_->add_category("notify", lokimq::Access{lokimq::AuthLevel::none, false, false})
         .add_command("add_pubkey", [this](auto& m) { this->handle_notify_add_pubkey(m); })
         .add_command("get_subscriber_count", [this](auto& m) { this->handle_notify_get_subscriber_count(m); });
+
+
+    lokimq_->add_category("service", lokimq::Access{lokimq::AuthLevel::none, false, false})

```

Comment:
Given that there are multiple commands here guarded by the same access key, you can make your life a little easier by using lokimq's access control; first change this line to:
```C++
lokimq_->add_category("service", lokimq::AuthLevel::admin)
```
and then when you call `listen_curve` give it an Allow callback that checks the key:
```C++
lokimq_->listen_curve(fmt::format("tcp://0.0.0.0:{}", port_),
        [this](std::string_view /*ip*/, std::string_view pk, bool /*sn*/) {
            return accesspk == access_control_key ? AuthLevel::admin : AuthLevel::none;
        });
```
and then you can delete all the manual key checking from the callbacks: the remote connection will get authenticated at connection time, and if they try to call anything inside the "service." category (which now requires admin access) they'll get a lokimq-level forbidden error if they didn't get recognized as admin during connection.